demos/classification_benchmark_demo/cpp/main.cpp

```diff
@@ -26,90 +26,93 @@
-    std::cout << "    -gt \"<path>\"              " << gt_message << std::endl;
-    std::cout << "    -d \"<device>\"             " << target_device_message << std::endl;
-    std::cout << "    -nthreads \"<integer>\"     " << num_threads_message << std::endl;
-    std::cout << "    -nstreams \"<integer>\"     " << num_streams_message << std::endl;
-    std::cout << "    -nireq \"<integer>\"        " << num_inf_req_message << std::endl;
-    std::cout << "    -nt \"<integer>\"           " << ntop_message << std::endl;
-    std::cout << "    -res \"<WxH>\"              " << image_grid_resolution_message << std::endl;
-    std::cout << "    -no_show                  " << no_show_message << std::endl;
-    std::cout << "    -time \"<integer>\"         " << execution_time_message << std::endl;
-    std::cout << "    -u                        " << utilization_monitors_message << std::endl;
-}
+namespace{
+constexpr char h_msg[] = "show the help message and exit";
+DEFINE_bool(h, false, h_msg);
+
+constexpr char i_msg[] = "an input to process. The input must be a single image, a folder of images";
+DEFINE_string(i, "", i_msg);
+
+constexpr char labels_msg[] = "path to .txt file with labels";
+DEFINE_string(labels, "", labels_msg);
+
+constexpr char m_msg[] = "path to an .xml file with a trained model";
+DEFINE_string(m, "", m_msg);
+
+constexpr char auto_resize_msg[] = "enables resizable input";
+DEFINE_bool(auto_resize, false, auto_resize_msg);
+
+constexpr char d_msg[] = "specify the target device to infer on. "
+    "The list of available devices is shown below. The demo will look for a suitable plugin for device specified. "
+    "Default value is CPU";
+DEFINE_string(d, "CPU", d_msg);
+
+constexpr char gt_msg[] = "path to ground truth .txt file";
+DEFINE_string(gt, "", gt_msg);
+
+constexpr char layout_msg[] = "specify inputs layouts."
+    "Ex. \"[NCHW]\" or \"input1[NCHW],input2[NC]\" in case of more than one input";
+DEFINE_string(layout, "", layout_msg);
+
+constexpr char nireq_msg[] = "number of infer requests";
+DEFINE_uint32(nireq, 0, nireq_msg);
+
+constexpr char nstreams_msg[] = "specify count of streams";
+DEFINE_string(nstreams, "", nstreams_msg);
+
+constexpr char nt_msg[] = "number of top results. Must be >= 1. Default value is 5";
+DEFINE_uint32(nt, 5, nt_msg);
+
+constexpr char nthreads_msg[] = "specify count of threads";
+DEFINE_uint32(nthreads, 0, nthreads_msg);
+
+constexpr char res_msg[] = "set image grid resolution in format WxH. Default value is 1280x720";
+DEFINE_string(res, "1280x720", res_msg);
+
+constexpr char show_msg[] = "disable showing of processed images";
+DEFINE_bool(show, true, show_msg);
+
+constexpr char time_msg[] = "time in seconds to execute program. Default is -1 (infinite time)";
+DEFINE_uint32(time, std::numeric_limits<gflags::uint32>::max(), time_msg);
+
+constexpr char u_msg[] = "list of monitors to show initially";
+DEFINE_string(u, "", u_msg);
```

Comment:
Thake the default value and wording from https://github.com/openvinotoolkit/open_model_zoo/blob/272d2d7001cefc712337f9cde2e2ca02b6b3c923/demos/interactive_face_detection_demo/cpp/main.cpp#L86

Reply:
Done